### PR TITLE
update readme and added return home link

### LINKED ISSUE #732: [qwik-graphql-tailwind] Add return to home button on the counter example page
# Description
- there needs to be a Return Home link at the bottom of the counter app page to allow easy navigation back to the home page

Image

Image

### PR DESCRIPTION
**Pull request by AllStackDev1**
## Type of change

- [x] Documentation change
- [ ] Bug fix

## Summary of change

Added return home link to counter page
Updated the readme doc

## Checklist

- [ ] These changes follow the [contributing guidelines](https://github.com/thisdot/starter.dev/blob/main/CONTRIBUTING.md)
- [ ] This starter kit has been approved by the maintainers
- [x] Changes for this new starter kit are being pushed to an integration branch instead of main
- [x] All dependencies are version locked
- [x] This fix resolves #732
- [x] I have verified the fix works and introduces no further errors

### CODE REVIEW

File: starters/qwik-graphql-tailwind/src/routes/counter-example/index.tsx [hunk 1]
```diff
@@ -10,6 +11,11 @@ export default component$(() => {
       <div className="py-5">
         <Counter />
       </div>
+      <div className="my-2.5">
```
Comment (SDaian) [inline starters/qwik-graphql-tailwind/src/routes/counter-example/index.tsx:14]: should we use `class` instead of `className`?

Comment (AllStackDev1) [reply starters/qwik-graphql-tailwind/src/routes/counter-example/index.tsx:14]: hmm, I think that will be a general update on the app. this ticket is more focus on the link

Comment (AllStackDev1) [reply starters/qwik-graphql-tailwind/src/routes/counter-example/index.tsx:14]: this has been resolved in a diff pr

**Review by SDaian — APPROVED** [review]
LGTM

**Pull request merged**